README.md

```diff
@@ -36,7 +36,7 @@ The initial migration will create the base data model as well as providing a way
 ```shell
 
 # Run migrations
-./maestro migrate
+./maestro migration
```

Comment:
can this revert to `migrate`?  that would keep it the same as every other ocm-like service.

Reply:
It was changed since we introduce the maestro-agent. refer to https://github.com/openshift-online/maestro/pull/20
In order to keep consistent, all the sub-commands use the noun e.g.:`server/agent/migration`. Does it make sense?